### ISSUE TITLE
Health/stats endpoint

### ISSUE THREAD

**Issue by kam193** [CybercentreCanada/assemblyline#187]
**Is your feature request related to a problem? Please describe.**
I'm looking into implementing some basic monitoring to ensure the overall system is in health. I observe some things like networking troubles etc. I'd like to have an API endpoint I can integrate with the monitoring system.
 
**Describe the solution you'd like**
The most basic and meaningful information would be to have an endpoint reporting the number of outstanding and queued submissions from Dispatcher and Ingester modules. Those give the best overall feeling, whether the system works or not.

**Describe alternatives you've considered**
1. The current `/system/status/ALL/` API is a great start, but it reports only if those modules are enabled, not how they work.
2. The Dashboard tab is a great place, but requires manual observation.
3. I assume this information could be extracted from Elasticsearch metrics, but it doesn't look like the best approach to get just basic info through dealing with deeply internal stuff.
4. More information - like error numbers, per-service stats - can be, of course, useful, but there is a reason why they are delivered using websocets, and not a typical API. A limited, basic info should be enough to indicate if there is a general problem or not.

**Additional context**
Dashboard tab is great, but it's always nice to connect with a centralized monitoring :)

**Comment by cccs-rs** [comment]
So would you be looking at an endpoint where you could get the data that's being pushed to the dashboard that you could poll periodically (as opposed to extracting them from Elasticsearch) ?

**Comment by kam193** [comment]
Exactly, then I could simply connect it to monitoring systems like Icinga/CheckMK. They are good at making HTTP requests, querying ES is a little different thing.

A great thing would be an overall status if all expected modules (e.g. scaler/dispatcher/ingestor) are up and running or if the dispatching/ingesting process was stopped.
